docs/source/installation/mac.rst

```diff
@@ -3,12 +3,22 @@ Mac
 
 First, you need to install Homebrew.  Homebrew is a package manager similar to
 ``apt`` or ``pacman`` on linux that allows you to install packages easily.  If
-it is not installed, please install it with this command:
+it is not installed, please install it with this command (for Apple Silicon useres, please read the tip first):
 
 .. code-block:: bash
 
    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" < /dev/null 2> /dev/null
 
+
+.. tip:: 
+
+    As of February 2020, the installation with ``brew`` has some issues with scipy and cairo packages in Apple Silicon mode. This can be solved by installing brew in the rosetta2 mode.
+    Therefore, a rosetta terminal needs to be created (see here: https://5balloons.info/correct-way-to-install-and-use-homebrew-on-m1-macs/). In case you installed ``brew`` in the normal terminal before, you have to make sure you are using the correct brew when installing packages. On default, the rosetta brew is located at ``/usr/local/bin/brew`` and the Apple Silicon brew is located at ``/opt/homebrew/bin/brew``(For more details on this, visit 
```

Comment:
```suggestion
    Therefore, a rosetta terminal needs to be created (see here: https://5balloons.info/correct-way-to-install-and-use-homebrew-on-m1-macs/). In case you installed ``brew`` in the normal terminal before, you have to make sure you are using the correct brew when installing packages. On default, the rosetta brew is located at "/usr/local/bin/brew" and the Apple Silicon brew is located at "/opt/homebrew/bin/brew" (For more details on this, visit 
```